### ISSUE TITLE
[5.1] script tag data-rules-json not rendered using vuejs

### ISSUE THREAD

**Issue by jeroenlammerts** [solspace/craft3-freeform#1233]
**Describe the bug or issue you're experiencing**
Having conditional logic applied to a form, a script tag is rendered before the form tag closes.
```
 $event->addChunk(
    '<script type="application/json" data-rules-json>'.$serialized.'</script>'
 );
```
When using vuejs 3 this causes the following warning, so the script tag is not rendered.
`[Vue warn]: Template compilation error: Tags with side effect (<script> and <style>) are ignored in client component templates.`

Freeform 4 is working in a different way using `data-ff-rule` attribute to fields.

**Steps to reproduce**

1. Build a site with vuejs 3 having a `<div id="app">` around the whole site.
2. Add conditional logic to a form
3. This is causing the error, the script tag is ignored.

**Expected behavior**
Maybe the plugin should respect the following setting and also insert the data-rules-json to that location.
```
Freeform Script Insertion Location
The location of where you want Freeform to insert its scripts for form and field functionality.
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.8.0
- Freeform Version: 5.1.14
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh install

**Comment by kjmartens** [comment]
Thanks for your feedback @jeroenlammerts.

I will get a developer to check into this and respond to you shortly. 🙂

**Comment by kjmartens** [comment]
Sorry for the delay @jeroenlammerts. This should now be resolved in Freeform 5.2.2. We've switched the `<script>` tags tp be `<div>` instead. 🙂
